Source/WebKit/WebProcess/Plugins/PDF/UnifiedPDF/UnifiedPDFPlugin.mm

```diff
@@ -2248,14 +2250,31 @@ static IntRect computeMarqueeSelectionRect(const WebCore::IntPoint& point1, cons
     return m_currentSelection.get().string;
 }
 
-bool UnifiedPDFPlugin::existingSelectionContainsPoint(const FloatPoint&) const
+bool UnifiedPDFPlugin::existingSelectionContainsPoint(const FloatPoint& rootViewPoint) const
 {
-    return false;
+    auto documentPoint = convertFromPluginToDocument(convertFromRootViewToPlugin(IntPoint { rootViewPoint }));
```

Comment:
Could make a helper for this

Reply:
Yeah I should probably just write out convertFromDocumentToRootView (and reverse)

Reply:
Fixed in https://github.com/WebKit/WebKit/commit/45f44bb919d81d06f4f846393bce791057252d49